Berksfile

```diff
@@ -2,3 +2,6 @@
 source 'https://supermarket.chef.io'
 
 metadata
+
+cookbook 'apt'
+cookbook 'yum'
```

Comment:
A simple/safe way of installing the git binary on linux is via the git cookbook:  

https://github.com/chef-cookbooks/git/blob/master/recipes/default.rb